src/app/components/AlertBox/index.tsx

```diff
@@ -61,12 +61,16 @@ export function AlertBox(props: Props) {
       background={{ color: mapStatus[props.status].background }}
       pad={{ horizontal: 'small' }}
     >
-      <Box direction="row" gap="small" align="center" justify={props.center ? 'center' : 'start'}>
-        {props.icon && <Text color={color}>{props.icon}</Text>}
-        <Text weight="bold" size="12px" style={{ marginTop: 10, marginBottom: 10 }}>
-          {props.children}
-        </Text>
-      </Box>
+      {!props.children && (props.icon || props.content) && (
```

Comment:
Would be useful to indicate this relation between content and children via types like 
```
type Props = {
  center?: boolean
  status: AlertBoxStatus
} & (
  | {
      content: React.ReactNode
      icon?: React.ReactNode
      children?: undefined
    }
  | {
      children: React.ReactNode
      content?: undefined
      icon?: undefined
    }
)
```